spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -1,43 +1,120 @@
 require 'spec_helper'
 
+describe Griddler::Cloudmailin::Adapter do
+  it 'registers itself with griddler' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.